src/speculators/convert/eagle/__main__.py

```diff
@@ -0,0 +1,8 @@
+"""
```

Comment:
Remove in favor of Python API entrypoints which are imported in root __main__.py file